smooch.go

```diff
@@ -182,32 +183,61 @@ func (sc *smoochClient) Send(userID string, message *Message) (*ResponsePayload,
 		return nil, err
 	}
 
-	req, err := http.NewRequest(http.MethodPost, url, buf)
+	var responsePayload ResponsePayload
+	err = sc.sendRequest(http.MethodPost, url, buf, &responsePayload)
 	if err != nil {
 		return nil, err
 	}
+
+	return &responsePayload, nil
+}
+
+func (sc *smoochClient) VerifyRequest(r *http.Request) bool {
```

Comment:
seems like this was just moved around and not a real change

Reply:
It the this change get detected. Moved all unexported function to the bottom.